src/components/App.js

```diff
@@ -49,6 +50,23 @@ const App = () => (
       <Privateroute exact path="/article/new" component={Create} />
       <Privateroute exact path="/article/:slug/edit" component={Editarticle} />
       <Privateroute exact path="/bookmarks" component={Bookmarks} />
+      <Route exact path="/articles" component={MyArticles} />
+      <Route exact path="/article/new" component={Create} />
+      <Route exact path="/article/:slug/edit" component={Editarticle} />
+      <Route exact path="/tags" component={ArticleTags} />
+      <Route
+        exact
+        path="/bookmarks"
```

Comment:
This is no longer needed since we created a private route to handle route protection.